chrome-extension/content.js

```diff
@@ -108,3 +115,23 @@ function customCreateElement(type, classList, innerHTML = '') {
   newElement.innerHTML = innerHTML;
   return newElement;
 }
+
+function addItem() {
+  newItem = {
+    sourceLink: 'www.googe.com', //TODO: get actual sourceLink
+    placesId: 'something', //TODO: get actual placesId, to-be implemented
```

Comment:
I think it's fine to just leave a todo for this placesId line, since it's an optional property and we haven't implemented it yet

Reply:
okay